recipe/meta.yaml

```diff
@@ -28,12 +28,9 @@ requirements:
   run:
 
 test:
+  requires:
+    - python <3.10
```

Comment:
Something is wrong with using python in the test environment. 
The error is `ModuleNotFoundError: No module named 'tzdata'`

Reply:
yes it seems python version and pkg  compatibility issue.